### PR TITLE
Score capability for five benchmark_eval_data products, and write the rubric they were banded against

### PR DESCRIPTION
**Pull request by ccerv1**
Calibration slice for issue #319. Five products scored, not the full 28, so the rubric application can be checked before the remaining 23 land in one tranche.

## The rubric, written down

`benchmark_eval_data` had no capability recipe. The category now declares one in its `scoring_recipe`, with `bands: null` and hand-authored rungs on `embeddings_retrieval`'s precedent, because the four dimensions do not combine by counting - GAIA is a 5 on a single domain and MT-Bench a 4 with judge scoring and no contamination control, so an attribute count reproduces neither.

The question is the one the issue names: does this benchmark still separate frontier models? Headroom leads, and contamination control, checkable scoring and distinctness say whether it will last.

- **5** - still separates frontier models and the design holds that: answers held out or the item set replaced over time, and an answer checkable without a judge.
- **4** - a real discriminator on a dimension nothing cheaper reaches, with one structural weakness: a judge, a small item set, or partial saturation.
- **3** - harder than the classic it replaced and still separating, but static and published in full.
- **2** - plateaued; a reporting convention rather than a discriminator.
- **1** - saturated, or shown not to measure what it claims.

The rungs reproduce all four existing bands: livebench, gaia and humanitys-last-exam at 5, mt-bench at 4. Three rules are recorded with them - saturation is a fact about the benchmark rather than a caveat about it, openness does not enter the band, and gating is not contamination control on its own (livecodebench is gated over an unstated license, math under a DMCA takedown).

## The five

**terminal-bench - 5**, basis `feature_matrix`.
Read the Terminal-Bench 2.0 paper (arXiv 2601.11868) and the repository README. The paper reports 89 tasks, each with its own environment, a human-written solution and comprehensive tests, and says frontier models and agents score under 65%. The README shows the per-task test script and oracle solution, the leaderboard pinned to a named dataset version, and the standing pointer at Terminal-Bench 2.0. Real headroom, graded by the task's own tests with no judge, and the suite is replaced version by version rather than frozen, which is how LiveBench holds this rung. The live leaderboard at tbench.ai was deliberately not cited: it renders its rows in the browser and its digest changed between two fetches minutes apart.

**swe-bench-verified - 4**, basis `feature_matrix`.
Read the HF dataset card (500 human-validated instances, "Evaluation is performed by unit test verification using post-PR behavior as the reference solution"), the SWE-bench paper, and the SWE-bench repository README. A real discriminator on a dimension nothing cheaper reaches, which is why frontier labs report against it. One rung below the top because the set has been frozen since August 2024 and the gold patch and tests ship with every instance, so its headroom is being spent rather than protected.

**mmlu-pro - 3**, basis `feature_matrix`.
Read the MMLU-Pro paper. Ten options instead of four, reasoning-focused items, noisy questions removed, a measured 16-33% accuracy drop against MMLU and prompt sensitivity down from 4-5% to 2%. Measurably more discriminating than what it replaced, and the claim is a measurement. It stays off the top two rungs because questions and answers ship together, nothing rotates and nothing is held back.

**mmlu - 2**, basis `feature_matrix`.
Read the original MMLU paper for the design and the MMLU-Pro paper for the plateau: "as models continue to improve, their performance on these benchmarks has begun to plateau, making it increasingly difficult to discern differences in model capabilities." The issue's own worked example. It survives as a reporting convention, which is the level-2 rung rather than an abstention.

**gsm8k - 1**, basis `feature_matrix`.
Read the GSM8K paper and GSM-Symbolic (arXiv 2410.05229), which reports that "the performance of all models declines when only the numerical values in the question are altered" and that one irrelevant added clause costs up to 65%. A score that moves on a rewrite preserving the problem is not a reading of mathematical reasoning, so GSM8K separates frontier models on neither difficulty nor reliability.

## The contamination instrument (added on review)

The Hugging Face API reports how many models declare a corpus as a training dataset, which is a countable, re-derivable input to the contamination-resistance dimension this recipe leads with:

```
https://huggingface.co/api/models?filter=dataset:<owner>/<name>&limit=1000&full=false
```

Readings across the category, taken 2026-09-11 and reproduced independently before recording:

| product | count | capability |
|---|---:|---:|
| livebench | 0 | 5 |
| terminal-bench | 0 | 5 |
| gaia | 5 | 5 |
| humanitys-last-exam | 11 | 5 |
| mt-bench | 1 | 4 |
| swe-bench-verified | 13 | 4 |
| humaneval | 29 | null |
| mmlu-pro | 30 | 3 |
| mmlu | 327 | 2 |
| gsm8k | 834 | 1 |

It reproduces all four pre-existing anchors and rank-orders the five new scores. It is declared under `contamination_control` in the recipe and added as corroborating evidence on each of the five records. **No score changed** — they stay 5/4/3/2/1.

**The signal is asymmetric, and that is the load-bearing part.** A high count is strong evidence that a benchmark has been trained on. A low count establishes nothing, because a corpus can reach a pretraining mix without any model tagging it on the Hub. `humaneval` is the proof: 29 models declare it and it is known-saturated on every other reading. So the instrument may move a band **down** and may never move one **up**, and it never overrides a read of the primary sources. Same rule as a transient fetch not being evidence of absence.

It is also one input to one of four dimensions rather than a rung of its own. `mt-bench` sits at 1 with a capability of 4, marked down for LLM-as-judge scoring and an 80-question set — facts about its design, not about contamination.

**The API, never the rendered page.** The two disagree: 834 against 1,109 for `openai/gsm8k`. The API figure is recorded because it is the one that re-fetches to a stable digest; the page is browser-rendered, the same class of source already avoided for tbench.ai. The discrepancy is written into the gsm8k record's `shows`.

## Published numbers move, and the issue predicted otherwise

The issue says the category stays at Stage 3 and nothing on the published map moves. It was already at **Stage 4**, and this change takes it to **Stage 3**.

`gsm8k` was the only fully-open product clearing the 4.5 maturity bar, and it cleared it on adoption 5 with a null capability. At the category's 0.6/0.4 weights it now blends to 3.4.

| | before | after |
|---|---|---|
| stage | 4, Competitive Open Ecosystem | 3, Viable Alternatives |
| gaps | `resiliency` | none |
| best fully-open | gsm8k, 5.0 | humaneval, 4.0 |

The `adoption` gap the issue expects does not fire either: the best fully-open product is now an adoption-only record at 4.0, which clears the adoption cutoff, so no driver gap fires. That may change once the remaining 23 land and the adoption-only records get capability scores of their own.

`notebook_data.json` and the notebooks are left to the regenerate bot.

## Open questions for review

- **`basis`.** All five use `feature_matrix`, matching the four anchors. An argument exists for `benchmark` here, since the reading behind the band is a published frontier figure on the product itself. Consistency with the anchors won; say if it should be the other way.
- **No peer edges.** Nothing records `relative_to`/`relation`. The rungs place the bands, and adding edges into a rubric under review seemed premature.
- **Rung distribution.** 9 of 32 scored, 4/2/1/1/1 across 5/4/3/2/1. The top rung is 44% of what is scored, which is not yet a distribution to judge - the 23 still unscored are mostly the saturated classics this rubric puts on 1 to 3.

## Gates

```
build.validate            0 error(s), 2 warnings (pre-existing model_families patterns)
build.check_verification  all four gates OK
build.check_capability    OK (12 weak roots pre-existing, report-only, none in this category)
build.check_recipe        0 failure(s)
build.check_rubric        benchmark_eval_data 30/30 reproduced, 2 deferred
build.check_freshness     all 20 categories within 45d
build.check_refetch --strict, each of the five: [OK] no fabricated digests
pytest -q                 1888 passed, 1 skipped (re-run after the instrument commit)
```

Every capability source on all five products, the training-tag counts included, re-fetched and reproduced its digest exactly. The drifts `check_refetch` reports on these products are on pre-existing openness and adoption sources (Hugging Face pages), not on anything added here.

Closes nothing yet - #319 stays open for the remaining 23.

### CODE REVIEW

**Comment by ccerv1** [comment]
Reviewed by Codex before closing, and the verdict is that closing unmerged is right on the evidence alone — independent of the direction change in #548. Five blocking defects: the contamination instrument's asymmetry rule holds only for declared use and cannot speak to the evaluation split; three of the five scores break that rule by treating a low count as evidence; the count query caps at limit=1000 so array length is not a total; gsm8k at 1 rests on a robustness finding that does not establish saturation, with no contemporary frontier accuracy cited; and the recipe is fitted to the existing scores rather than defining reproducible boundaries.

Full findings and the salvage guidance are on #546, so the work stays findable. Nothing here is wasted — the research dimensions and the cited evidence are worth keeping, with narrower wording.

**Comment by ccerv1** [comment]
Closing unmerged, as agreed. The direction this explored was set aside in #548, and Codex review found the slice's own evidence would not have supported merging it regardless — five blocking defects, recorded on #546 along with what is worth salvaging.

Nothing here is wasted. The research dimensions, the cited evidence and the contamination-instrument idea all survive on #546 with narrower wording; the branch stays available if anyone reopens the question.
